README.md

```diff
@@ -1,26 +1,90 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+//
```

Comment:
This is logging your number from 1 to -10. Had you started at 10 the cut off could be 1, and you could put one condition in a while loop.